authomatic/core.py

```diff
@@ -50,8 +50,8 @@ def normalize_dict(dict_):
 
     """
 
-    return dict([(k, v[0] if not isinstance(v, str) and len(v) == 1 else v)
-                 for k, v in list(dict_.items())])
+    return {[(k, v[0] if not isinstance(v, str) and len(v) == 1 else v)
+             for k, v in list(dict_.items())]}
```

Comment:
I think it could be rewritten like this:

```suggestion
    return {k: v[0] if not isinstance(v, str) and len(v) == 1 else v for k, v in dict_.items()}
```